docs/docs/configure-task-handling.md

```diff
@@ -0,0 +1,18 @@
+---
+Configuring Task Handling
+---
+
+You can configure Marathon's actions on unreachable tasks. The `unreachableStrategy` parameter of your app definition allows you to configure this in two ways: by defining when a new task instance should be launched, and by defining when a task instance should be expunged.
+
+```
+"unreachableStrategy": {
+	"timeUntilInactiveSeconds": <integer>,
+	"timeUntilExpungeSeconds": <integer>
+}
+```
+
+- `timeUntilInactiveSeconds`: If a task instance is unreachable for longer than this value, it is marked inactive and a new instance will launch. At this point, the unreachable task is not yet expunged. The minimum value for this parameter is 1.
+
+- `timeUntilExpungeSeconds`: If an instance is unreachable for longer than this value, it will be expunged. An expunged task will be killed if it ever comes back. Instances are usually marked as unreachable before they are expunged, but that is not required. The minimum value for this parameter is 2.
```

Comment:
It has been renamed to `unreachableExpungeAfterSeconds`.

"The minimum value for this parameter is 2." That is not quite true. It has to be bigger than `unreachableInactiveAfterSeconds`. The rest is good : ).